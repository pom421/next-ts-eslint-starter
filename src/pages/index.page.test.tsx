import Home from "@/pages/index.page"
import { render } from "@testing-library/react"

test("render as expected", () => {
  render(<Home />)

  expect(document.body.innerHTML).toMatchSnapshot()
})
