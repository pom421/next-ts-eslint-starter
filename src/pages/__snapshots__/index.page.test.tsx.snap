// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render as expected 1`] = `"<div><div class=\\"container\\"><main class=\\"main\\"><h1>Next TS ESLint starter</h1><div><p>You have clicked 0 times</p><button>Click here</button></div></main><footer class=\\"footer\\"><a href=\\"https://vercel.com?utm_source=create-next-app&amp;utm_medium=default-template&amp;utm_campaign=create-next-app\\" target=\\"_blank\\" rel=\\"noopener noreferrer\\">Powered by <span class=\\"logo\\"><span style=\\"box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;\\"><span style=\\"box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;\\"><img style=\\"display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;\\" alt=\\"\\" aria-hidden=\\"true\\" src=\\"data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%2772%27%20height=%2716%27/%3e\\"></span><img alt=\\"Vercel Logo\\" src=\\"data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7\\" decoding=\\"async\\" data-nimg=\\"intrinsic\\" style=\\"position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;\\"><noscript></noscript></span></span></a></footer></div></div>"`;
